new_platforms/samples/oe_policy_tool/host/host.cpp

```diff
@@ -0,0 +1,477 @@
+            result,
+            oe_result_str(result));
+        throw EOTHER;
+    }
+    auto enclave_destructor =
+        finally([&enclave] { oe_terminate_enclave(enclave); });
+
+    size_t report_size = 0;
+    uint8_t* report;
+    result = oe_get_report_v2(enclave, 0, NULL, 0, &report, &report_size);
+    if (result != OE_OK)
+    {
+        fprintf(
+            stderr,
+            "oe_get_report(): result=%u (%s)\n",
+            result,
+            oe_result_str(result));
+        throw EOTHER;
+    }
+    auto report_destructor = finally([&report] { oe_free_report(report); });
+
+    {
+        std::ofstream report_file(output_path);
+        if (!report_file)
+        {
+            throw EOTHER;
+        }
+        report_file << std::string((char*)report, report_size);
+    }
+}
+
+std::vector<uint8_t> get_file_contents(const std::string& filename)
+{
+    std::ifstream in(filename);
+    if (!in)
+    {
+        throw EOTHER;
+    }
+
+    std::vector<uint8_t> contents;
+    in.seekg(0, std::ios::end);
+    contents.resize((size_t)in.tellg());
+    in.seekg(0, std::ios::beg);
+    in.read((char*)&contents.front(), contents.size());
+    in.close();
+    return contents;
+}
+
+oe_report_t append_report(const std::string& filename)
+{
+    std::vector<uint8_t> report = get_file_contents(filename);
+
+    oe_report_t parsed_report;
+    oe_result_t result =
+        oe_parse_report(&report.front(), report.size(), &parsed_report);
+    if (result != OE_OK)
+    {
+        throw EOTHER;
+    }
+
+    return parsed_report;
+}
+
+void iterate_over_pem_files(
+    const std::string& source_directory,
+    std::function<void(const std::string&)> action)
+{
+    const char* filename;
+
+#ifdef WIN32
+    HANDLE handle = INVALID_HANDLE_VALUE;
+    WIN32_FIND_DATA file_data;
+
+    const std::string source_mask = source_directory + PEM_FILES;
+
+    handle = FindFirstFile(source_mask.c_str(), &file_data);
+    if (handle == INVALID_HANDLE_VALUE)
+    {
+        throw EOTHER;
+    }
+    auto handle_destructor = finally([&handle] { FindClose(handle); });
+
+    do
+    {
+        filename = file_data.cFileName;
+
+        action(filename);
+    } while (FindNextFile(handle, &file_data));
+#endif
+#ifdef LINUX
+    DIR* d = NULL;
+    struct dirent* dir = NULL;
+
+    d = opendir(source_directory.c_str());
+    if (d == NULL)
+    {
+        throw EOTHER;
+    }
+    auto d_destructor([&d] { closedir(d); });
+
+    while ((dir = readdir(d)) != NULL)
+    {
+        if (strcmp(dir->d_name, ".") == 0 || strcmp(dir->d_name, "..") == 0)
+        {
+            continue;
+        }
+        filename = dir->d_name;
+
+        action(filename);
+    }
+#endif
+}
+
+std::vector<oe_report_t> extract_reports(const std::string& source_directory)
+{
+    std::vector<oe_report_t> reports;
+
+    auto action = [&reports, &source_directory](const std::string& filename) {
+        const std::string fullpath = source_directory + "\\" + filename;
```

Comment:
In line 381 you have Linux code, but Linux doesn't use "\\" as the path separator

Reply:
Changed to "/" in the other repo.